index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="/css/reset.css" />
+    <link rel="stylesheet " href="/css/header.css" />
+    <link rel="stylesheet " href="/css/main.css" />
+    <link rel="stylesheet " href="/css/footer.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="header">
```

Comment:
💊 제안
이미 header라는 시맨틱 태그를 사용하고 계신데, 내부에 다시 header라는 클래스명을 쓰신 건 중복적이라 가독성이 떨어질 수 있습니다.
header 태그는 그 자체로 의미가 드러나기 때문에, 클래스명은 역할이나 스타일링 목적을 드러내는 이름으로 바꾸시는 것을 추천드립니다!

```suggestion
    <header class="header">
      <div class="header-inner">
```